rig-dockerfile-talk/main.slide

```diff
@@ -0,0 +1,55 @@
+docker images and rig
+a story about images in the cloud
+
+Jon Morehouse
+Erick Yellott
+
+* What is rig?
+
+a workflow for building and deploying container images
+
+:hopefully everyone has heard of rig by now. It's our approach to deploying containerized applications on AWS
+:rig is responsible for taking our pull-requests, identifying a Dockerfile to build and building that dockerfile into the registry so we can go ahead and deploy it at will to various environments
+
+* What is Docker?
+
+- tool for building images
+- tool for uploading images
+- tool for running containers from an image
+
+plus a lot more (its a big tool!)
+
+* How does rig use Docker?
+
+- rig is a wrapper around Amazon's ECS
+- ECS is a way to run containers
+
+* What's a Docker Image?
+
+- a set of immutable layers
+- snapshot of a "container"
+
+* Docker Registries
+
+- central image store
+- custom protocol for storing / fetching
+
+* the Rig workflow
+
+- github pings builder
+- builder is responsible for running `rig build`
+- this creates a final "image" and uploads it to `ecs`
```

Comment:
should we mention ECR, and mention that's why we need to enter our gpg password on `vagrant up`?